boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -95,6 +95,15 @@
 		};
 	};
 
+	nrf_radio_coex: nrf7002-coex {
+		status = "okay";
+		compatible = "nordic,nrf700x-coex";
+		req-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
+		status0-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+		grant-gpios = <&gpio0 24 GPIO_ACTIVE_HIGH>;
+		swctrl1-gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
+	};
+
```

Comment:
We do not support the 'old generic three pin interface (GPIO)' for nrf5340.  It's only for 52840.  I think it's better to leave out the GPIO implementation.

Reply:
From NCS 2.1 the old/new generic 3 pin interface is supported (in experimental state) for both nRF 52 and nRF53 with SDC.
Before NCS 2.1, there were no support for generic 3 pin with SDC.

Reply:
Ok, so anything to do in this matter?

Reply:
Yes, we do